servicetalk-client-api/src/main/java/io/servicetalk/client/api/AbstractRetryingFilterBuilder.java

```diff
@@ -287,7 +294,7 @@ private ReadOnlyRetryableSettings(final int maxRetries,
             this.initialDelay = initialDelay;
```

Comment:
Perhaps range check this here as well?

Reply:
same as above I would prefer to keep validation in a single spot to avoid duplicating logic (and getting out of date).

Reply:
Ok, the suggestion was about validating at creation/build vs usage. Build time validation should usually be preferred where possible because the usage may be temporally and organizationally disconnected from the error and thus harder to diagnose and resolve.